src/deepsparse/server/server.py

```diff
@@ -121,6 +121,7 @@ def _build_app(server_config: ServerConfig) -> FastAPI:
 
     app = FastAPI()
     server_logger = build_logger(server_config)
+    system_logging_manager = SystemLoggingManager.from_server_config(server_config)
```

Comment:
This is WIP to make the tests pass. I will focus more on E2E testing in the next PRs.